Description: Uncomments keystone_authtoken directives
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-03-04

--- designate-2.0.0~b3.orig/etc/designate/designate.conf.sample
+++ designate-2.0.0~b3/etc/designate/designate.conf.sample
@@ -154,12 +154,12 @@ debug = False
 # Keystone Middleware
 #-----------------------
 [keystone_authtoken]
-#auth_host = 127.0.0.1
-#auth_port = 35357
-#auth_protocol = http
-#admin_tenant_name = service
-#admin_user = designate
-#admin_password = designate
+auth_host = 127.0.0.1
+auth_port = 35357
+auth_protocol = http
+admin_tenant_name = service
+admin_user = designate
+admin_password = designate
 #memcached_servers = localhost:11211
 
 #-----------------------
