Description: Set enable-api-v2 to True by default
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/831696
Bug-Ubuntu: https://bugs.launchpad.net/designate/+bug/1604043
Forwarded: not-needed
Last-Update: 2016-07-16

--- designate-2.0.0.orig/designate/api/__init__.py
+++ designate-2.0.0/designate/api/__init__.py
@@ -38,7 +38,7 @@ cfg.CONF.register_opts([
                help='The strategy to use for auth. Supports noauth or '
                     'keystone'),
     cfg.BoolOpt('enable-api-v1', default=True),
-    cfg.BoolOpt('enable-api-v2', default=False),
+    cfg.BoolOpt('enable-api-v2', default=True),
     cfg.BoolOpt('enable-api-admin', default=False),
     cfg.IntOpt('max_header_line', default=16384,
                help="Maximum line size of message headers to be accepted. "
